### PR TITLE
testing/libzookeeper: new aport

### PR DESCRIPTION
**Pull request by gilek**
https://zookeeper.apache.org
The Apache ZooKeeper system for distributed coordination is a high-performance service for building distributed applications.

### CODE REVIEW

File: testing/libzookeeper/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+pkgrel=0
+pkgdesc="The Apache ZooKeeper system for distributed coordination is a high-performance service for building distributed applications."
+url="https://zookeeper.apache.org"
+arch="all"
+license="Apache License"
+depends=""
+makedepends=""
+install=""
+subpackages="$pkgname-dev"
+source="ftp://ftp.task.gda.pl/pub/www/apache/dist/zookeeper/zookeeper-$pkgver/zookeeper-$pkgver.tar.gz"
```
Comment (Ikke) [inline testing/libzookeeper/APKBUILD]: Wouldn't it be better to use an [HTTP](https://www.apache.org/dyn/closer.cgi#http) mirror here? For example travis does not allow downloads via FTP.

File: testing/libzookeeper/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	./configure --prefix=/usr
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+}
+
+dev() {
```
Comment (Ikke) [inline testing/libzookeeper/APKBUILD]: Isn't this handled by `default_dev`? Or am I missing something here.

Comment (jirutka) [reply testing/libzookeeper/APKBUILD]: Yes, this should be handled by default dev…

File: testing/libzookeeper/APKBUILD (hunk 1 above)
Comment (jirutka) [reply testing/libzookeeper/APKBUILD]: Definitely, FTP should not be used in abuilds.

**Pull request closed without merging**